scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -213,6 +213,14 @@ function New-DockerNATNetwork {
     Write-Output "Created customnat network with flag: com.docker.network.windowsshim.disable_gatewaydns=true"
 }
 
+function Disable-DockerDefaultNATNetwork {
+    . "$SCRIPTS_DIR\scripts\variables.ps1"
+    Stop-Service $DOCKER_SERVICE_NAME
+    Get-HNSNetwork | Remove-HNSNetwork
+    Set-Content -Path "${DOCKER_DATA}\config\daemon.json" -Value '{ "bridge" : "none" }' -Encoding Ascii
```

Comment:
This code seems to assume that daemon.json didn't exist before this line and a new daemon.json file will contain only this new bridge settings. 
There is Update-Docker call right after this. Would be it better to the call the Disable-DockerDefaultNATNetwork call after Update-Docker in case that Update-Docker got changed (eg, to reinstall Docker instead of just copying binaries)

Reply:
That's correct. This assumes that `daemon.json` doesn't exist already. But all our DC/OS Azure images have a default Docker installation (which doesn't generate any `daemon.json`), so we should be fine for now. In the future, I'll make sure I take into consideration if the file already exists.

Regarding when to call `Disable-DockerDefaultNATNetwork`: We should be fine for now because we require Docker to be pre-installed. In case we take into consideration that Docker is not installed, this line should be called after we install Docker.

Thank-you for your comment,
Ionut